src/core/primitives/Avatar/fragments/AvatarPrimitiveImage.tsx

```
@@ -19,9 +19,10 @@ const AvatarPrimitiveImage = ({
     const { handleErrorImage, handleLoadImage, hasError } = useContext(AvatarPrimitiveContext);
 
     // If there's no src or there's an error, render nothing
-    if (!src || hasError) {
+    if (hasError) {
         return null;
     }
+    useEffect(()=>{if(!src) handleErrorImage()},[])
```

ESLint

[error] Requires a space after '{'.

(block-spacing)